src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -645,14 +648,9 @@ private StreamReader GetFileStreamReader(Uri uri, out Encoding encoding)
             encoding = null;
```

Comment:
Nit: You don't have to initialize `encoding`, because the call to `GetContentStream` will assign it. The compiler checks to make sure that `out` parameters are assigned on all code paths.

Reply:
OK